src/content/troubleshooting/_index.en.md

```diff
@@ -0,0 +1,136 @@
+           fallthrough =====> proceed as usual if lighthouse can't resolve
+        }
+```
+If the entries highlighted above are missing, it means CoreDNS wasn't configured with lighthouse. It can be easily added by running `kubectl edit configmap coredns` and making the changes manually. You may need to repeat this step on every cluster.
+
+##### Check submariner-lighthouse-agent
+Next we check if `submariner-lighthouse-agent` is running correctly or not. Run `kubectl -n submariner-operator get pods submariner-lighthouse-agent` and check status of pods.
+
+If status is error with `ImagePullBackOff`, run `kubectl -n submariner-operator describe deployment submariner-lighthouse-agent` and check if `Image` is set correctly to `quay.io/submariner/lighthouse-agent:<version>`. If it is and still getting the same error, raise an issue as image has likely gone missing from quay.io
+
+If status is just error, run `kubectl -n submariner-operator get pods` to get the pod name for lighthouse agent. Then run `kubectl -n submariner-operator logs <lighthouse-agent-pod-name>` to get the logs. See if there are any errors in the log. If yes, raise an [issue](https://github.com/submariner-io/lighthouse/issues) with log contents, or you can continue reading through document to troubleshoot this further.
+
+If there are no errors, grep the logs for service name that you're trying to query, we may need them later for raising an issue.
+
+##### Check multiclusterservice CRs
+If all this was fine, next we need to check if `multiclusterservices` CRs were created correctly for the service you're trying to access. The format of `multiclusterservice` object's name is as follows:
+
+`<service-name>-<service-namespace>-<cluster-id>`
+
+Run `kubectl get multiclusterservices --all-namespaces |grep <your-service-name>` on the broker cluster to see if CR was created for your service or not. If not, check the Lighthouse Agent logs on the cluster where service was created, there should be some error or warning messages about being unable to create `multiclusterservice` object for your service. Most common error can be `Forbidden` if the RBAC wasn't configured correctly. Depending on deployment used, *subctl* or *helm* it should've been done for you. Create an [issue](https://github.com/submariner-io/lighthouse/issues) with relevant log entries.
+
+If the CR was created correctly on broker cluster, next step is to check for it on the cluster where you're trying to access the service. Follow the same steps as earlier to get list of `multiclusterservices` objects created and if your service name shows up in it or not. If not, again check the logs of Lighthouse Agent but this time on the cluster where you are trying to access the service. Again, it will likely be an issue with RBAC and you follow the same steps, create an Create an [issue](https://github.com/submariner-io/lighthouse/issues) with relevant log entries.
```

Comment:
```suggestion
If the CR was created correctly on broker cluster, next step is to check for it on the cluster where you're trying to access the service. Follow the same steps as earlier to get list of `multiclusterservices` objects created and if your service name shows up in it or not. If not, again check the logs of Lighthouse Agent but this time on the cluster where you are trying to access the service. Again, it will likely be an issue with RBAC and you follow the same steps, create an [issue](https://github.com/submariner-io/lighthouse/issues) with relevant log entries.
```